libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -264,7 +264,15 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("PTCH_FF_K", 30, AP_TECS, _pitch_ff_k, 0.0),
 
-    // 31 previously used by TECS_LAND_PTRIM
+    // 31 previously used by AP_Int8 TECS_LAND_PTRIM which was removed in November 2022
+
+    // @Param: THR_ERATE
+    // @DisplayName: Forward throttle external limit slew rate
+    // @Description: The forward throttle lower limit controlled by TECS when set externally will be reduced at this rate. Set to a non positive value to hold the lower limit for one frame only
+    // @Units: ms
```

Comment:
I don't think it is in milliseconds, it is proportion per second. So the default of 0.2 means decay over 5 seconds. I think we either need a better description, or make it in seconds and call it TECS_THR_DECTIME for "decay time".

Reply:
I'll change it to percentage per second and update the description.